### DISCUSSION TITLE
仓库拆分

### DISCUSSION THREAD

**Discussion by zaydenyang** [Tencent/libpag#115] [Ideas] [answered]
是否考虑按照各端拆分成小仓库呢？比如 libpag-ios/libpag-android，这直接上来clone就 1.48G了

**Comment by kevingpqi123** [comment, marked as answer]
如果你只是运行demo，可以通过下面的仓库：
https://github.com/libpag/pag-ios
https://github.com/libpag/pag-android

**Reply by domchen** [reply]
这个并不是平台多导致的仓库大，是跨平台的公共测试文件会比较大。如果拆开多个仓库，会导致每个仓库都是1.48G。
